config/action.d/pf.conf

```diff
@@ -91,5 +90,5 @@ allports = any
 
 # Option: multiport
 # Notes.: addition to block access only to specific ports
-# Usage.: use in jail config: "banaction = pf[actiontype=<multiport>]"
+# Usage.: use in jail config: "banaction = pf[actiontype=<multiport>, port="%(port)s"]"
```

Comment:
>  > -# Usage.: use in jail config: "banaction = pf[actiontype=<multiport>]"
>  > +# Usage.: use in jail config: "banaction = pf[actiontype=<multiport>, port="%(port)s"]"
>    You are wrong here: the port will be specified through action_ or action,
>    but no longer with banaction.

yes -- you are right!  didn't spot that it was for banaction and not for
action.  I will revert this change, thanks!